src/monster.cpp

```diff
@@ -1480,6 +1483,46 @@ bool monster::move_effects( bool )
 
     bool u_see_me = g->u.sees( *this );
     if( has_effect( effect_tied ) ) {
+        // friendly pet, will stay tied down and obey.
+        if( friendly == -1 ) {
+            return false;
+        } else {
```

Comment:
```suggestion
        }
```
And than you can remove one level of indentation below.